src/defaultsettings.cpp

```diff
@@ -76,11 +76,19 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("tooltip_show_delay", "400");
 
 	// Some (temporary) keys for debugging
+	#ifdef NDEBUG
```

Comment:
I'd prefer `DEBUG` here, and swapping the two blocks, just to keep from having an unnecessary negative. It's also more consistent with the above (`keymap_toggle_update_camera`)